dataactvalidator/validation_handlers/validator.py

```diff
@@ -449,29 +474,35 @@ def cleanSplit(string, toLower = True):
         return stringList
 
     @staticmethod
-    def getMultiValues(rule,record):
+    def getMultiValues(rule,record,interfaces):
         """ Create string out of values for all fields involved in this rule
 
         Args:
             rule: Rule to return fields for
             record: Record to pull values from
-
+            interfaces: InterfaceHolder for DBs
         Returns:
             One string including all values involved in this rule check
         """
         ruletext1 = Validator.cleanSplit(rule.rule_text_1, True)
         ruletext2 = Validator.cleanSplit(rule.rule_text_2, True)
-        fields = ruletext1 + ruletext2 #TODO try catch
+        ruleType = interfaces.validationDb.getMultiFieldRuleTypeById(rule.multi_field_rule_type_id)
+        if ruleType == "CAR_MATCH" or ruleType == "REQUIRED_CONDITIONAL" or ruleType == "REQUIRE_ONE_OF_SET":
```

Comment:
Hmm, this could be added as an extra column to the rule_type table that defines what fields to show values for.  I'll see if I can include that in my current refactor of the Rule tables